javascript/aws-lambda/security/tainted-html-response.yaml

```diff
@@ -0,0 +1,42 @@
+rules:
+- id: tainted-html-response
+  severity: WARNING
+  message: >-
+    Detected user input flowing into an HTML response. You may be
+    accidentally bypassing secure methods
+    of rendering HTML by manually constructing HTML and this could create a cross-site
+    scripting vulnerability, which could let attackers steal sensitive user data.
+  metadata:
+    cwe:
+      "CWE-79: Improper Neutralization of Input During Web Page Generation ('Cross-site Scripting')"
+    owasp:
+      - A07:2017
+      - A03:2021
+    category: security
+    technology:
+      - aws-lambda
+  languages:
+  - javascript
+  - typescript
+  mode: taint
+  pattern-sources:
+  - patterns:
+    - pattern-either:
+      - pattern-inside: |
+          exports.handler = function ($EVENT, ...) {
+            ...
+          }
+      - pattern-inside: |
+          function $FUNC ($EVENT, ...) {...}
+          ...
+          exports.handler = $FUNC
+      - pattern-inside: |
+          $FUNC = function ($EVENT, ...) {...}
+          ...
+          exports.handler = $FUNC
+    - pattern: $EVENT
+  pattern-sinks:
```

Comment:
We might want to add sanitizers for known Javascript HTML sanitizer libraries like DOMPurify

Reply:
agree, I'll test this rule on real targets and try to collect list of popular sanitizers